src/components/SchemaTable/index.css

```diff
@@ -0,0 +1,20 @@
+* {
+  box-sizing: 0;
+  margin: 0;
+  padding: 0;
+}

```

Comment:
You can check to see if it would make sense for us to add `CSSBaseline` https://material-ui.com/components/css-baseline/ It provides good defaults.

Reply:
That's a great idea. Applied changes!